### PR TITLE
added login page

### PR DESCRIPTION
**Pull request by DivyanshSareen**
preview: 
image

### CODE REVIEW

File: login.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+/* login section */
+
+.login-area{
+    display: flex;
+    justify-content: center;
+    width: 100%;
+}
+
+form{
```
Comment (rutu14) [inline login.css:63]: Suggestion - Can use a class selector, so that your other forms are not changed.

File: login.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                </div>
+            </div>
+        </div>
+    </nav>
+    <section class="login-area">
+        <form action="">
+            <h5 class="h5 center-text login-head">Login</h5>
+            <div class="input">
+                <label for="email" class="input-label">Email address</label>
+                <input type="email" class="input-data" placeholder="example@gmail.com" required>
```
Comment (rutu14) [inline login.html:46]: ID Attribute - You missed out the ID attribute for email input.

File: login.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            </div>
+            <div class="options">
+                <div class="input remember-check">
+                    <input type="checkbox" class="input-data" id="remember">
+                    <label for="remember">Remember me</label>
+                </div>
+                <div class="paragraph2 forgot-password">Forgot your Password?</div>
+            </div>
+            <div class="btn btn-login center-text">Login</div>
+            <div class="paragraph1 center-text">Create New Account <i class="fa-solid fa-chevron-right"></i></div>
```
Comment (rutu14) [inline login.html:60]: Suggestion - This can be a link button with anchor tag.

File: login.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+            <div class="input">
+                <label for="password" class="input-label">Password</label>
+                <input type="password" id="password" class="input-data" placeholder="********" required>
+            </div>
+            <div class="options">
+                <div class="input remember-check">
+                    <input type="checkbox" class="input-data" id="remember">
+                    <label for="remember">Remember me</label>
+                </div>
+                <div class="paragraph2 forgot-password">Forgot your Password?</div>
```
Comment (rutu14) [inline login.html:57]: Suggestion - This can be a link button with anchor tag.

**Review by rutu14 — APPROVED** [review]
UI Suggestion

- The name on the navbar looks smaller, can be made a little large.  
- The login button can have a larger font size.
- The search icon on focus on input disappears, so on input focus the search icon can have different color.
- The spacing between the login text and it's box is too big, can go with some smaller spacing.

UI Appreciation : The colors and overall look is nice.

File: login.html (hunk 4 above)
Comment (DivyanshSareen) [reply login.html:57]: Thank you for the suggestions, will make sure to cover these in future PRs

**Review by sanjitsarkar — APPROVED** [review]
Everything looks good you can fix the contrast issue with the button

**Pull request merged**